### DISCUSSION TITLE
Father's Day

### DISCUSSION THREAD

**Discussion by bjlockie** [FossifyOrg/General-Discussion#474] [Q&A]
Why don't events like Father's Day show up in Calendar?
They show in Google Calendar so where are they coming from?
They aren't "Holidays".

**Comment by Aga-C** [comment]
Holiday lists for the app are created manually by contributors, and these lists contain only public and bank holidays. In the future they will be created automatically, but still only with public and bank holidays.

> They show in Google Calendar so where are they coming from?

Google Calendar has its own list, and it's not related to our app.
